fabbwled-backend/src/main/resources/sections/397-432.yaml

```diff
@@ -50,6 +50,20 @@
           then:
             - turnTo: 250
 
+- number: 418
+  content:
+    - text: |
```

Comment:
`|` keeps newlines (`>` would not), see https://yaml-multiline.info/
...don't think we need a newline after "be" mid-sentence for example :)